src/components/CorpusIndex.js

```diff
@@ -153,20 +153,22 @@ const CorpusIndex = ({data}) => {
         {
           props => (
             <div>
-              <SearchBar {...props.searchProps}/>
-              {(data.description || data.license) && (
-                <div>
+                <div className="corpus-description">
```

Comment:
If you change the structure of the markup you should reindent accordingly. I changed that.